frontends/p4/fromv1.0/converters.cpp

```diff
@@ -399,7 +399,20 @@ ExternConverter::convertExternCall(const IR::Declaration_Instance *ext, const IR
     return new IR::MethodCallStatement(prim->srcInfo, mc);
 }
 
+std::map<cstring, ExternConverter *> *ExternConverter::cvtForType = nullptr;
 
+void ExternConverter::addConverter(cstring type, ExternConverter *cvt) {
```

Comment:
What is type? Extern name or extern type name? How qualified? It would be helpful to add a comment and an example.